ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +407,173 @@ def test_trustdomain_disable_disables_subdomain(self):
+        assert 'Group name: child1' in cmd_output.stdout_text
+        assert 'Member groups: parent' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child2', '--groups', 'child1'])
+        assert 'Group name: child2' in cmd_output.stdout_text
+        assert 'Member groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member groups: parent' in cmd_output.stdout_text
+
+        # add user to group 'parent'
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'parent', '--users', username])
+        assert 'Group name: parent' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member of groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member of group: child2' in cmd_output.stdout_text
+
+        # clear sssd_cache
+        clear_sssd_cache(master)
+
+        # user lookup
+        cmd_output = master.run_command(['ipa', 'user-show', username])
+        assert 'parent' in cmd_output.stdout_text
+        assert 'ipausers' in cmd_output.stdout_text
+        assert 'child1' in cmd_output.stdout_text
+        assert 'child2' in cmd_output.stdout_text
+
+        clear_sssd_cache(master)
+
+        sshcon = paramiko.SSHClient()
+        sshcon.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+        try:
+            cmd = 'ssh -i {} -q {}@{} groups'.format(user_ssh_priv_key,
+                                                     username, client.hostname)
+            cmd_output = master.run_command(cmd)
+            assert '{}'.format(username) in cmd_output.stdout_text
+            assert 'parent' in cmd_output.stdout_text
+            assert 'child1' in cmd_output.stdout_text
+            assert 'child2' in cmd_output.stdout_text
+        except (paramiko.AuthenticationException,
+                paramiko.SSHException) as e:
+            pytest.fail('Authentication using SSH key not successful', e)
+
+        # add member
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child1', '--users', username])
+        assert 'Group name: child1' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member groups: parent' in cmd_output.stdout_text
+        assert 'Member of groups: child2' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        # now check ssh on the client
+        clear_sssd_cache(client)
+        try:
+            cmd = 'ssh -i {} -q {}@{} groups'.format(user_ssh_priv_key,
+                                                     username, client.hostname)
+            cmd_output = client.run_command(cmd)
```

Comment:
Also need to copy the user_ssh_priv_key file to the client.

Reply:
added the copy to master and client part